src/controllers/__tests__/item.test.js

```diff
@@ -351,4 +339,58 @@ describe('Items Endpoints', () => {
       expect(responseBody.message).toBe('Requested item ID is not valid!');
     });
   });
+
+  describe('PUT /api/items/:id', () => {
+    test('Should update name on matching item', async () => {
+      const response = await request(server)
+        .put(`/api/items/${itemId}`)
+        .send({ name: 'Couch' });
+
+      expect(response.header['content-type']).toContain('application/json');
+      expect(response.statusCode).toBe(200);
+      expect(response.body.name).toEqual('Couch');
+    });
+
+    test('Should update isAvailable field on matching item', async () => {
+      const response = await request(server)
+        .put(`/api/items/${itemId}`)
+        .send({ isAvailable: false });
+
+      expect(response.header['content-type']).toContain('application/json');
+      expect(response.statusCode).toBe(200);
+      expect(response.body.isAvailable).toEqual(false);
+    });
+
+    test('Should response with an error message when to be updated item ID not found', async () => {
+      const response = await request(server)
+        .put('/api/items/6208e47a5fe21cc475419234')
+        .send({ isAvailable: false });
+
+      expect(response.header['content-type']).toContain('application/json');
+      expect(response.statusCode).toBe(422);
+      expect(response.body.message).toBe(
+        'The item with the specified ID was not found.'
+      );
+    });
+  });
+
+  describe('DELETE /api/items/:id', () => {
+    test('Should delete matching item', async () => {
+      const response = await request(server).delete(`/api/items/${itemId}`);
+
+      expect(response.statusCode).toBe(204);
```

Comment:
For a stronger test, you can execute a GET request on this `itemId` and expect it to be not found.

Because otherwise this test is just checking if response code is 204, but it does not check if the item actually got deleted on the DB.